apps/wallet/src/ui/app/pages/site-connect/index.tsx

```diff
@@ -88,16 +87,16 @@ function SiteConnectPage() {
 						rejectTitle="Reject"
 						onSubmit={handleHideWarning}
 						isWarning
-						addressHidden
+						address={activeAddress}
```

Comment:
I think showing the active address here as connected might be a bit confusing. At the time we show this warning the users doesn't have any connected accounts with the dapp and can chose later the accounts that will connect with